src/planning.py

```diff
@@ -423,6 +424,65 @@ def _run_low_level_search(task: Task, option_model: _OptionModelBase,
     return [], True
 
 
+def _run_plan_with_option_model(
+        task: Task, option_model: _OptionModelBase, plan: List[_Option],
+        timeout: float) -> Tuple[LowLevelTrajectory, bool]:  # pragma: no cover
+    """Runs a plan on an option model to generate a low level trajectory.
+
+    Returns a LowLevelTrajectory and a boolean. If the boolean is True,
+    the option sequence successfully executed to achieve the goal and
+    generated a LowLevelTrajectory. Otherwise, it returns and empty list
+    and False.
+    """
+    start_time = time.time()
+    traj: List[State] = [task.init] + [DefaultState for _ in plan]
+    actions: List[Action] = [Action(np.array([0.0])) for _ in plan]
+    for idx in range(len(plan)):
+        if time.time() - start_time > timeout:
+            return LowLevelTrajectory([task.init], [], False), False
+        state = traj[idx]
+        option = plan[idx]
+        cur_idx = idx + 1
+        if option.initiable(state):
```

Comment:
instead of this `if` and `else`, do:
```
if not option.initiable(state):
    # The option is not initiable.
    return LowLevelTrajectory([task.init], [], False), False
<code that's currently in the if block>
```

this will let you unindent all this code inside here by one level